### DISCUSSION TITLE
Acrobat: NVDA shortcut to 'escape' from the UI and start browsing the document proper?

### DISCUSSION THREAD

**Discussion by medeaoblongata** [nvaccess/nvda#14117] [Q&A]
I'm exploring NVDA+Acrobat. I've had some modest success, but I keep getting 'trapped' in the UI.

Is there a shortcut which will reliably put me in browse mode in the main document view? Sometimes it just works, but more often it doesn't. Clicking in the main view does nothing interesting.

Related question: Should NVDA behave the same with both Acrobat Reader and Acrobat Pro (regarding the currently open document)?

**Comment by Brian1Gaff** [comment]
I've never used pro, but I'd have thought so.
 PDF files do seem to be a very mixed bag though, many seem to read far 
better in Edge than they do in Adobes own reader, which seems crazy.
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message ----- 
From: "Brennan Young" ***@***.***>
To: "nvaccess/nvda" ***@***.***>
Cc: "Subscribed" ***@***.***>
Sent: Thursday, September 08, 2022 11:03 AM
Subject: [nvaccess/nvda] Acrobat: NVDA shortcut to 'escape' from the UI and 
start browsing the document proper? (Issue #14115)

**Comment by cary-rowen** [comment]
Hi, can you find the document control if you press F6?

**Comment by medeaoblongata** [comment]
Had to wait a while before the problem reappeared. 
F6 seems to do something (not sure what), but didn't solve the problem. Closing and reopening the PDF did work in this case (but not always).
